contribute/quality-assurance.rst

```diff
@@ -20,9 +20,9 @@ If you find two issues describing the same problem, leave a comment and try to f
 Testing patches
 ---------------
 
-Pull-requests can be tested using the `QA scripts <https://github.com/ubports/ubports-qa-scripts>`_. Run ``ubports-qa -h`` for usage information.
+Pull-requests can be tested using the `QA scripts <https://gitlab.com/ubports/development/core/ubports-qa-scripts>`_. Run ``ubports-qa -h`` for usage information.
 
-Once the pull-request has been merged, the issue it fixes is moved to the quality assurance column of the `GitHub project <https://github.com/ubports/ubuntu-touch/projects/3>`__. Please check if the issues in this column are still present in the latest update on the devel channel, then see if anything else has broken in the update. Check if the developer mentioned specific things to look out for when testing and leave a comment detailing your experience. If you have write-access to the repository, you can move the issue back to **In Development** (and reopen it) or forward to **Release Candidate** as specified by the :doc:`issue tracking guidelines </about/process/issue-tracking>`.
+Once the pull-request has been merged, the issue it fixes is moved to the completed column of the `Gitlab Milestone <https://gitlab.com/groups/ubports/development/core/-/milestones>`__. Please check if the issues in this column are still present in the latest update on the devel channel, then see if anything else has broken in the update. Check if the developer mentioned specific things to look out for when testing and leave a comment detailing your experience. If you have write-access to the repository, you can move the issue back to **In Development** (and reopen it) or forward to **Release Candidate** as specified by the :doc:`issue tracking guidelines </about/process/issue-tracking>`.
```

Comment:
Are you sure the Milestones are a drop-in replacement for the Github project? They don't seem to be actively used.

Reply:
Whoops, wrong link; i had this lying around since I forgot to PR it in a while ago somehow, and it seems like the milestones are no longer under `development/core`; changed the link to show all milestones in the `ubports` GitLab account.

Reply:
That looks better :)